Source/Particles/Deposition/ChargeDeposition.H

```diff
@@ -90,22 +114,31 @@ void doChargeDepositionShapeN (const GetParticlePosition& GetPosition,
     }
 #endif
 
+    const auto dxiarr = geom.InvCellSizeArray();
+    const auto plo = geom.ProbLoArray();
+    const auto domain = geom.Domain();
+
+    const amrex::Box sample_tbox(IntVect(AMREX_D_DECL(0, 0, 0)), a_tbox_max_size - 1);
+    amrex::Box sample_tbox_x = convert(sample_tbox, ix_type);
+
+    sample_tbox_x.grow(depos_order);
```

Comment:
For the out of bounds access, I tried adding a margin of up to `+8` here and in the corresponding `.grow()` call in the kernel, but it still fails on the negative values.

Reply:
Andrew's fixes handled this